build_scripts/build_libtiff.sh

```diff
@@ -19,6 +19,7 @@ pushd third_party/libtiff
 patch -p1 < ${ROOT_DIR}/patches/0001-Fix-wget-complaing-about-expired-git.savannah.gnu.or.patch
 patch -p1 < ${ROOT_DIR}/patches/libtiff-CVE-2022-22844.patch
 patch -p1 < ${ROOT_DIR}/patches/libtiff-CVE-2022-0561.patch
+patch -p1 < ${ROOT_DIR}/patches/libtiff-CVE-2022-0626.patch
```

Comment:
Why `libtiff-CVE-2022-0626.patch` is not included in this PR?

Reply:
I missed that. Fixed.